index.html

```diff
@@ -196,53 +174,57 @@ <h3 id="resource_wide_defaults">Resource-wide defaults</h3>
 
 
 <div class="req" id="bp_not_only_default">
-<p class="advisement">Specifications MUST NOT assume that a document-level default is sufficient.</p>
+<p class="advisement">Specifications MUST NOT assume that a document-level default is sufficient.  Even If a resource-wide setting is available, specify field-based metadata to override the default.</p>
```

Comment:
This BP uses the term "document-level default" but this section uses the terminology "resource-wide". Suggest we use "resource-wide" in this best practices.

Reply:
Done.